src/Umbraco.Web/Redirects/RedirectUrlSearchResults.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using Umbraco.Core.Models;
+
+namespace Umbraco.Web.Redirects
+{
+    public class RedirectUrlSearchResult
+    {
+        public string StatusMessage { get; set; }
```

Comment:
Yep, I'll wait for @madsrasmussen to do his magic and then we can adjust the casing of this.